tests/licensedcode/data/licenses_reference_reporting/license-reference-works-with-clues.expected.json

```diff
@@ -3,430 +3,80 @@
     {
       "identifier": "python-c496ccae-69a0-c5f8-a742-83bcd66bfe68",
       "license_expression": "python",
-      "count": 1,
-      "detection_log": [
-        "not-combined"
-      ],
-      "matches": [
-        {
-          "score": 100.0,
-          "start_line": 23,
-          "end_line": 26,
-          "matched_length": 35,
-          "match_coverage": 100.0,
-          "matcher": "2-aho",
-          "license_expression": "python",
-          "rule_identifier": "python_not_not-a-license_269.RULE",
-          "rule_url": "https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/rules/python_not_not-a-license_269.RULE"
-        }
-      ]
+      "detection_count": 1,
+      "detection_log": []
```

Comment:
I think you may want this removed from top level unless requested.

Reply:
@pombredanne false alarm :P This was added because we had `--license-diagnostics` enabled here. 
See for example:

-  a test with this disabled (i.e. not used) [test file](https://github.com/nexB/scancode-toolkit/blob/license-detection-follow-up/tests/licensedcode/test_licenses_reference.py#L32)  [result file](https://github.com/nexB/scancode-toolkit/blob/license-detection-follow-up/tests/licensedcode/data/licenses_reference_reporting/scan-with-reference.expected.json#L83) 

-  a test with this enabled [test_file](https://github.com/nexB/scancode-toolkit/blob/license-detection-follow-up/tests/licensedcode/test_plugin_license_detection.py#L66) [result_file](https://github.com/nexB/scancode-toolkit/blob/license-detection-follow-up/tests/licensedcode/data/plugin_license/unknown_intro/scan-unknown-intro-dual-license.expected.json#L73)